models/iosxr/vrf_global/iosxr_vrf_global.yaml

```diff
@@ -0,0 +1,86 @@
+module: iosxr_vrf_global
+short_description: Manages global VRF configuration.
+description:
+  - This module manages VRF configurations on Cisco IOS-XR devices. It enables playbooks to handle either individual VRFs or the complete VRF collection. It also permits removing non-explicitly stated VRF definitions from the setup.
+version_added: 9.0.0
+author: Ruchi Pakhle (@Ruchip16)
+notes:
+  - Tested against Cisco IOSXR Version 9.0.0
+  - This module works with connection C(network_cli). See L(the IOS_XR Platform Options,../network/user_guide/platform_iosxr.html)
+  - For more information on using Ansible to manage network devices see the :ref:`Ansible Network Guide <network_guide>`
+  - For more information on using Ansible to manage Cisco devices see the `Cisco integration page <https://www.ansible.com/integrations/networks/cisco>`_.
+options:
+  config:
+    description: A dictionary of options for VRF configurations.
+    type: dict
+    suboptions:
+      name:
```

Comment:
I think we need to add a top level key called vrfs under with it would be a list of dicts based on the conversation we had in the channel.

Reply:
that was specifically for ios i believe, should we do that for iosxr, i am not sure coz we dont have other stuff to include here.

Reply:
My bad, got confused with ios